pypeit/scripts/show_2dspec.py

```diff
@@ -8,17 +8,25 @@
 This script enables the viewing of a processed FITS file
 with extras.  Run above the Science/ folder.
 """
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-from __future__ import unicode_literals
 
 import argparse
 from astropy.table import Table
 from pypeit import ginga
 from pypeit.spectrographs import util
 from pypeit.processimages import ProcessImagesBitMask as bitmask
 from pypeit.core import pixels
+import os
+import numpy as np
+import IPython
```

Comment:
remove `IPython`

Reply:
Note the script has an option to embed after running in case the user wants to inspect some property of the images. IPython is thus a dependency.

Reply:
Ok.  Add to the install notes and 
.travis and requirements.txt.

Reply:
Sounds good, can you instruct me on how to do that?

Reply:
Add a link to iPython in the `installing.rst` file
Like the others.

Add a line in the `requirements.txt` file. Like the others
I.e.  copy+paste